### ISSUE TITLE
Migrate Default's into Action Configuration YAML?

### ISSUE THREAD

**Issue by bsieber-mozilla** [mozilla/jira-bugzilla-integration#20]
The following lines establish defaults for a a few of the action config fields:
https://github.com/mozilla/jira-bugzilla-integration/blob/d5f84bc90b5e4fe1fd3172f71c9df38b5b3a901b/src/jbi/configuration.py#L47-L49

Perhaps a new section in the config.*.yaml could be added to control this from configuration.
https://github.com/mozilla/jira-bugzilla-integration/blob/d5f84bc90b5e4fe1fd3172f71c9df38b5b3a901b/config/config.dev.yaml#L1-L5

_Should there be a new section? Maybe there's an alternate route to explore here_

Should this new section be: `default`, `default.actions`, `actions._default`, or `actions.default`.
- [ ] `default`: as a top-level section
- [ ] `default.actions`: a top-level section that has a section dedicated to the other top-level section `action`
- [ ] `actions._default`: adding the underscore should ensure yamllint keeps this element at the top of the list (we can also remove ordering of keys from yamllint config)
- [ ] `actions.default` 🤷

**Comment by bsieber-mozilla** [comment]
Not something we need to act on at this time; but--time-permitting--I'd love your thoughts on this: @grahamalama @leplatrem

**Comment by grahamalama** [comment]
This might also be a thing we could use pydantic for:
```python
class ActionConfig(BaseModel, extra=Extra.allow):
    action: str = "src.jbi.whiteboard_actions.default"
    enabled: bool = False
    parameters: dict = {}
```

**Comment by leplatrem** [comment]
I like the idea to rely on pydantic models

**Comment by leplatrem** [comment]
Implemented in #28
